### ISSUE TITLE
Running the output files gives segmentation fault

### ISSUE THREAD

**Issue by nihalar** [nitrieu/frigatev2#1]
When I try to run with the command "./frigate ./circuits/aes/p32_aes.wir -dp" I get segmentation fault error.

**Comment by nitrieu** [comment]
Hi, please check out https://github.com/AarhusCrypto/DUPLO for the final version.

**Comment by nihalar** [comment]
> Hi, please check out https://github.com/AarhusCrypto/DUPLO for the final version.

Oh thanks. I'm searching for where the compiled circuit is located, could you please help me with the same?

**Comment by nihalar** [comment]
> Hi, please check out https://github.com/AarhusCrypto/DUPLO for the final version.

Thank You,
Does frigate support float?

**Comment by nitrieu** [comment]
no, integer only :( but you can convert the float to integer by shift and truncation

**Comment by nihalar** [comment]
> no, integer only :( but you can convert the float to integer by shift and truncation

#parties 2

typedef int_t 32 int

#input 1 int
#input 2 int
#output 1 int


function void main()
{
 output = input1 + input2;
}

trying to compile this gives me a segmentation fault. Could you look into it?

**Comment by nitrieu** [comment]
did you check out the frigate version from DUPLO?
